### ISSUE TITLE
NO log file (0KB in size), file is empty

### ISSUE THREAD

**Issue by Bob-Collins** [lisamelton/other_video_transcoding#211]
I recently upgraded my laptop and am now running Windows 11.  I am able to run the tools to transcode-video successfully, but the log files are empty.  at the end of the encode, I am getting the following output from the transcode:

"Encode done!

HandBrake has exited.
C:/Users/xxxx/AppData/Local/Microsoft/WindowsApps/transcode-video: invalid byte sequence in UTF-8

C:\Out>"

Why is this happening now?  I am not a programmer, and I just fell back to Ruby 3.2 instead of Ruby 3.3, and the error still occurs.

Thanks in advance!

**Comment by lisamelton** [comment]
@Bob-Collins I'm sorry you're having this problem. I don't know enough about your system configuration to properly diagnose this. If I had to guess, I would say your RubyGems setup is borked. I suggest uninstalling the Gem and then reinstalling it.

The main thing I'm confused about is which Gem and tool you're talking about. You've filed a GitHub issue on the `other_video_transcoding` project and Gem which contains the `other-transcode` tool. But you identify the `transcode-video` tool which is not in that project. That tool is part of the older `video_transcoding` project and Gem. Sooooo... which Gem and which tool are you talking about?

**Comment by Bob-Collins** [comment]
I honestly do not know then, I want to use the new method, as I know the other one is older and outdated.  I followed these instructions to install
Installation

My Video Transcoding tools are designed to work on macOS, Linux and Windows. They're packaged as a Gem and require Ruby version 2.0 or later. See "Installing Ruby" if you don't have the proper version on your platform.

Use this command to install the package:

gem install video_transcoding

You may need to prefix that command with sudo in some environments:

sudo gem install video_transcoding

Updating

Use this command, or the variation prefixed with sudo, to update the package:

gem update video_transcoding

Requirements

Most of the tools in this package require other software to function properly, specifically these command line programs:


	HandBrakeCLI
	ffmpeg
	mkvpropedit
	mp4track


Previewing the output of detect-crop is optional, but doing so uses mpv, a free, Open Source, and cross-platform media player.

You can download the command line version of HandBrake, called HandBrakeCLI, here:

https://handbrake.fr/downloads2.php

On macOS, HandBrakeCLI and all its other dependencies can be easily installed via Homebrew, an add-on package manager:

brew install handbrake

brew install ffmpeg

brew install mkvtoolnix

brew install mp4v2

The optional crop previewing package can also be installed via Homebrew:

brew install mpv

On Linux, package management systems vary so it's best consult the indexes for those systems. But there's a Homebrew port available called Linuxbrew and it doesn't require root access.

On Windows, it's best to follow one of the two methods, manually installing binaries or installing into the Windows Subsystem for Linux, as described here:

https://github.com/JMoVS/installing_video_transcoding_on_windows

When installing HandBrakeCLI or other downloaded programs, make sure the executable binary is in a directory listed in your PATH environment variable. On Unix-style systems like macOS and Linux, that directory might be /usr/local/bin.

If you're comfortable using Docker virtualization software, a pre-built container with everything you need, plus installation instructions, is available here:

https://hub.docker.com/r/ntodd/video-transcoding/


If that is incorrect, I would need to know where to find the correct instructions for installation.

Apologies for my lack of knowledge, but I am not a programmer, and used the older method for several years..

Thanks in advance!
Bob Collinsworth
 
 



On Tue, 03 Dec 2024 09:31:42 -0800, Lisa Melton ***@***.***> wrote:
 
 

@Bob-Collins I'm sorry you're having this problem. I don't know enough about your system configuration to properly diagnose this. If I had to guess, I would say your RubyGems setup is borked. I suggest uninstalling the Gem and then reinstalling it.

The main thing I'm confused about is which Gem and tool you're talking about. You've filed a GitHub issue on the other_video_transcoding project and Gem which contains the other-transcode tool. But you identify the transcode-video tool which is not in that project. That tool is part of the older video_transcoding project and Gem. Sooooo... which Gem and which tool are you talking about?

**Comment by lisamelton** [comment]
@Bob-Collins No problem. You don't need to be a programmer to use the tools. Just to be able to use the command line. And you seem more than capable of that. You're good.

OK, to use the new tool then you need to download the `other-transcode.rb` script and install in manually since it's not a Gem anymore.

Please see the instructions on the Other Video Transcoding `README.md` file here:

https://github.com/lisamelton/other_video_transcoding/blob/master/README.md

And read the Installation instructions here:

https://github.com/lisamelton/other_video_transcoding/blob/master/README.md#installation

**Comment by Bob-Collins** [comment]
OK, I have it working, but I am not getting an output log file like the older project did, where the log file was written in the same directory with the output file.  In addition, I would like to be able to change the audio output from DTS or DD5.1 down to stereo and add gain on it so I am not constantly having to adjust the volume on my TV.  How can I do that with the new method?  in the old I could always use this command line:

transcode-video --filter detelecine --audio-width main=stereo --handbrake-option gain=6.0 "\in\Big Bang Theory - S11E09 - The Bitcoin Entanglement.mkv"

Do I no longer need to use the detelecine option as well as the audio-width, and the gain option?  I also used to be able to force the framerate in the older method, because I had some videos where it used to come out "jagged" in terms of panning shots.

Any help would be so much appreciated.

Thanks in advance!
Bob Collinsworth
 


On Tue, 03 Dec 2024 10:08:34 -0800, Lisa Melton ***@***.***> wrote:
 
 

@Bob-Collins No problem. You don't need to be a programmer to use the tools. Just to be able to use the command line. And you seem more than capable of that. You're good.

OK, to use the new tool then you need to download the other-transcode.rb script and install in manually since it's not a Gem anymore.

Please see the instructions on the Other Video Transcoding README.md file here:

https://github.com/lisamelton/other_video_transcoding/blob/master/README.md

And read the Installation instructions here:

https://github.com/lisamelton/other_video_transcoding/blob/master/README.md#installation

**Comment by lisamelton** [comment]
@Bob-Collins For the `other-transcode.rb` script, a `.log` file is now only created when the `--debug` option is used.

Please look at the output of `other-transcode.rb --help full` to learn about its options because they are _completely_ different from `transcode-video`.

Also, please consult the Wiki for more information here:

https://github.com/lisamelton/other_video_transcoding/wiki

**Comment by Bob-Collins** [comment]
Yes,  I can see that the options have indeed changed.  But one I see that appears to be missing is how to increase the loudness of the audio, in the old version I would use the follow sequence to increase the loudness by 6db:

--handbrake-option gain=6.0

How would I do the same thing with the new method, or is this not possible anymore?

Bob Collinsworth
 


On Tue, 03 Dec 2024 11:28:29 -0800, Lisa Melton ***@***.***> wrote:
 
 

@Bob-Collins For the other-transcode.rb script, a .log file is now only created when the --debug option is used.

Please look at the output of other-transcode.rb --help full to learn about its options because they are completely different from transcode-video.

Also, please consult the Wiki for more information here:

https://github.com/lisamelton/other_video_transcoding/wiki

**Comment by Bob-Collins** [comment]
I read the Wiki, and yes,  I can see that the options have indeed changed.  But one I see that appears to be missing is how to increase the loudness of the audio, in the old version I would use the follow sequence to increase the loudness by 6db:

--handbrake-option gain=6.0

How would I do the same thing with the new method, or is this not possible anymore?

Bob Collinsworth


On Tue, 03 Dec 2024 11:28:29 -0800, Lisa Melton ***@***.***> wrote:
 
 

@Bob-Collins For the other-transcode.rb script, a .log file is now only created when the --debug option is used.

Please look at the output of other-transcode.rb --help full to learn about its options because they are completely different from transcode-video.

Also, please consult the Wiki for more information here:

https://github.com/lisamelton/other_video_transcoding/wiki
